cosmoz-treenode-navigator.html

```diff
@@ -41,7 +45,24 @@
 			}
 
 			iron-list {
-				max-height: var(--cosmoz-treenode-navigator-max-list-height, 50vh);;
+				height: 100vh;
+				max-height: var(--cosmoz-treenode-navigator-max-list-height, 50vh);
```

Comment:
I think `height: 100vh` will always be constrained by the `max-height` directive.
Actually, I don't think we need `max-height` here, but only `height`.
Something like 
```
height: var(--cosmoz-treenode-navigator-list-height, 50vh);
```
That would allow users to size the height of the list.